### PR TITLE
Hello, world!

### PR DESCRIPTION
**Pull request by bartek7**
Translated "Hello World" page into polish language

### CODE REVIEW

File: 1-js/02-first-steps/01-hello-world/1-hello-alert/task.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
-importance: 5
+Trudność: 5
```
Comment (Verthon) [inline 1-js/02-first-steps/01-hello-world/1-hello-alert/task.md]: In main repo I found this `Please don't translate "importance" (and other top metadata).`

File: 1-js/02-first-steps/01-hello-world/2-hello-alert-ext/task.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-importance: 5
+Trudność: 5
```
Comment (Verthon) [inline 1-js/02-first-steps/01-hello-world/2-hello-alert-ext/task.md]: In main repo I found this `Please don't translate "importance" (and other top metadata).`

File: 1-js/02-first-steps/01-hello-world/1-hello-alert/task.md [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-# Show an alert
+# Pokaż alert
 
-Create a page that shows a message "I'm JavaScript!".
+Utwórz alert na stronie z treścią "Jestem JavaScript!".
 
-Do it in a sandbox, or on your hard drive, doesn't matter, just ensure that it works.
-
-[demo src="solution"]
+Zrób do gdziekolwiek np. w sandbox lub na dysku twardym. Upewnij się, że działa.
```
Comment (Verthon) [inline 1-js/02-first-steps/01-hello-world/1-hello-alert/task.md]: Typo Zrób `do` gdziekolwiek

File: 1-js/02-first-steps/01-hello-world/article.md [hunk 4]
```diff
@@ -92,29 +91,29 @@ To attach several scripts, use multiple tags:
...
-Other pages that reference the same script will take it from the cache instead of downloading it, so the file is actually downloaded only once.
+Jeśli inne strony muszą skorzystać z tych skryptów, a plik został już pobrany, robią to z pamięci podręcznej, zamiast pobierać go ponownie.
 
-That reduces traffic and makes pages faster.
+To znacznie przyspiesza działanie strony i redukuje niepotrzebny ruch. 
 ```
 
-````warn header="If `src` is set, the script content is ignored."
-A single `<script>` tag can't have both the `src` attribute and code inside.
+````warn header="Jeśli atrybut `src` jest dodany, wtedy treść w tym tagu nie jest wykonywany."
```
Comment (Verthon) [inline 1-js/02-first-steps/01-hello-world/article.md]: What do you think about: "Jeśli atrybut `src` jest dodany, wtedy skrypt nie jest wykonywany"?

Comment (bartek7) [reply 1-js/02-first-steps/01-hello-world/article.md]: thanks

File: 1-js/02-first-steps/01-hello-world/2-hello-alert-ext/task.md (hunk 2 above)
Comment (bartek7) [reply 1-js/02-first-steps/01-hello-world/2-hello-alert-ext/task.md]: thanks

**Comment by jakubdrozdek** [comment]
Outdated and out of sync. Had some translation issues, so I translated it from scratch and put in another PR.

**Pull request closed without merging**